moc_openstack/manifests/ssl/add_cinder_cert.pp

```diff
@@ -0,0 +1,23 @@
+class moc_openstack::ssl::add_cinder_cert (
+  $cinder_crt      = "/etc/pki/tls/certs/cinder.crt",
+  $cinder_key      = "/etc/pki/tls/private/cinder.key",
+  $cinder_crt_path = "puppet:///modules/quickstack/certs/host.crt",
```

Comment:
You should move it to moc_openstack. We don't want to put any of our moc specific files in the quickstack module.